cspell/.cspell.json

```
@@ -70,6 +70,7 @@
         "planing"
     ],
     "words": [
+        "Kaida"
```

```suggestion
        "Kaida",
```
Need the trailing comma.